feat: add OpenResty demo running on docker (#1)
DataDome/datadome-openresty-demo: README.md

# DataDome OpenResty demo

> This repo tests the DataDome OpenResty module using docker.

Reviewer: ```suggestion
This repository provides an example of the DataDome OpenResty module using Docker.
```

> For additional information, please check our [public documentation](https://docs.datadome.co/docs/openresty)

Reviewer: ```suggestion
For additional information, please check our [public documentation](https://docs.datadome.co/docs/openresty).
```

Reviewer 2: picky... (@Lauredg )

## 1 - Configuration

> Open file `nginx.conf` and set your `sever-side-key`

Reviewer: ```suggestion
Open the file `nginx.conf` and set your `server-side-key` (found inside your [DataDome Dashboard](https://app.datadome.co/dashboard/management/integrations)).
```

> Open file `nginx.conf` and set your `sever-side-key`

Reviewer 2: ```suggestion
Open file `nginx.conf` and set your `datadome-sever-side-key`
```

Author: updating it, thanks!

## 2 - Build

```
docker image build . -t my-datadome-openresty-image
```

## 3 - Run

```
docker run -d --rm --name myDataDomeOpenResty -p 8282:80 my-datadome-openresty-image
```

## 4 - Test

```
curl -v  http://localhost:8282/
```

> You should be able to see DataDome cookie

Reviewer: ```suggestion
Your request was protected (the header `X-DataDome: protected` is present) and is shown in your Dashboard.
```

> You should be able to see DataDome cookie

Reviewer 2: ```suggestion
DataDome cookie is present
Request shows in the DataDome dashboard
```

Author: just updated with @Lauredg 's suggestion, thanks
